src/buildings/building.gd

```diff
@@ -39,14 +39,35 @@ func _ready():
 	set_original_color()
 	build_timer_ui.visible = false
 	TickManager.tick.connect(_on_tick)
-	ResourceManager.upgrade_acquired.connect(apply_upgrades)
-	apply_upgrades()
+	ResourceManager.upgrade_acquired.connect(check_for_adjacency_multiplier)
+	check_for_adjacency_multiplier()
 
 	if placed and building_complete:
 		_setup_scan_for_nearby_bonus()
 
+func _process(delta):
```

Comment:
Does moving this further up fix a load order issue? Or is it just a preference? No change required, just curious.

Reply:
It just preference, since stuff in _process are tend to affect performance the most so if it near the top we can check if more frequently if it need optimization